docs/tutorials/compute_for_lambda.rst

```diff
@@ -0,0 +1,314 @@
+<https://docs.globus.org/api/flows/getting-started/>`_. See also the documentation on
+the `Compute Action Provider for Flows <../actionprovider>`_.
+
+
+.. _compute_for_lambda_dependency_management:
+
+Dependency Management
+=====================
+
+Since Compute functions contain no dependency information beside ``import`` statements,
+endpoint administrators must take care to ensure any needed dependencies are present on
+the endpoint side. For static workflows a manual approach is sufficient---ie,
+administrators install packages as needed to the Python environment that the endpoint
+is running in---but there are mechanisms that allow for more dynamic situations.
+
+These lean on Compute's :ref:`configuration template <user-config-template-yaml-j2>`
+capabilities, which don't have an analog in Lambda. In short, Compute endpoints spawn
+child processes that actually handle the execution of the task, and the configuration
+of these child processes is determined by a combination of the endpoint's configuration
+template and the details of the task at hand.
+
+One approach allows users to :doc:`specify a container
+<../tutorials/dynamic_containers>` with their task. With the following configuration
+options included:
+
+.. code-block:: yaml+jinja
+   :caption: ``user_config_template.yaml.j2`` with dynamic containers
+   :emphasize-lines: 4-6
+
+   display_name: My Containerized Endpoint
+   engine:
+     type: GlobusComputeEngine
+     container_type: {{ container_type }}
+     container_uri: {{ container_uri }}
+     container_cmd_options: {{ container_cmd_options|default() }}
+     provider:
+       init_blocks: 1
+       max_blocks: 1
+       min_blocks: 0
+       type: LocalProvider
+
+Users can then leverage the ``user_endpoint_config`` feature of the executor to choose
+a container:
+
+.. code-block:: python
+  :caption: Submitting with a container
+
+   from globus_compute_sdk import Executor
+
+   ep_id = "..."
+   config = {
+      "container_type": "docker", # or podman, apptainer...
+      "container_uri": "<my docker container>",
+   }
+
+   with Executor(
+      endpoint_id=ep_id,
+      user_endpoint_config=config,
+   ) as gcx:
+      gcx.submit(...)
+
+    # or:
+
+    with Executor(endpoint_id=ep_id) as gcx:
+      gcx.user_endpoint_config = config
+      gcx.submit(...)
+
+.. important::
+  The container being used must have the ``globus-compute-endpoint`` package installed.
+
+For static workflows where users always invoke the same function, administrators can
+simply install dependencies directly into the endpoint's Python environment. In these
+cases administrators might want to ensure that **only** this known function is called,
+which can be achieved with a :ref:`function allow list <function-allowlist>`:
+
+.. code-block:: yaml
+  :caption: :ref:`config.yaml <endpoint-manager-config>` with a function allow list
+
+  allowed_functions:
+    - eca16448-bfe2-445a-900c-cd46b1426522
```

Comment:
We're developers, not award-winning essayists, but I do observe that this article ends rather abruptly.